applications_tests/gls_navier_stokes_2d/gls_static_droplet_surface_tension_force.prm

```diff
@@ -51,7 +51,7 @@ subsection analytical solution
   set verbosity = verbose
     subsection uvwp
     	set Function constants = sigma=0.023, R=0.022
-    	set Function expression = 0; 0; if ((x-0.04) * (x-0.04) + (y-0.04) * (y-0.04) < 0.02 * 0.02 , 2 * sigma / R , 0)
+    	set Function expression = 0; 0; if ((x-0.04) * (x-0.04) + (y-0.04) * (y-0.04) < 0.02 * 0.02 , sigma / R , 0)
```

Comment:
There was an error in the analytical solution? Have you looked into convergence?